src/pages/Guilds/CreateProposal.tsx

```diff
@@ -16,20 +22,169 @@ const SidebarContent = styled(Box)`
   @media only screen and (min-width: 768px) {
     margin-left: 1rem;
   }
+  @media only screen and (max-width: 768px) {
+    margin-top: 1rem;
+  }
 `;
 
 const PageContent = styled(Box)`
   @media only screen and (min-width: 768px) {
     margin-right: 1rem;
   }
 `;
+
+const Button = styled(IconButton)`
+  margin: 0;
+`;
+
+const SidebarHeader = styled.h3`
+  font-family: Inter;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 24px;
+  color: #000000;
+  padding: 16px;
+  margin: 0;
+`;
+
+const Label = styled.span`
+  font-family: Inter;
+  font-style: normal;
+  font-weight: 500;
+  font-size: ${({ size }) => (size ? size : `14px`)};
+  line-height: 20px;
+  display: flex;
+  color: ${({ color }) => (color ? color : `#000000`)};
+  margin-bottom: 0.5rem;
+`;
+
 const CreateProposalPage: React.FC = () => {
+  const history = useHistory();
+  const [editMode, setEditMode] = React.useState(true);
+  const [title, setTitle] = React.useState('');
+  const [referenceLink, setReferenceLink] = React.useState('');
+  const [proposalBodyHTML, setProposalBodyHTML] = React.useState('');

```

Comment:
I feel like we should move the `useLocalStorageWithExpiry` hooks we currently have in the editor to here, and pass the state down to the editor component as props. Right now we're duplicating the states in both places.

Reply:
Ok. Refactored to use useLocalStorageWithExpiry hook on create proposal page.